docs/using-radicle/creating-patches.md

```diff
@@ -0,0 +1,97 @@
+---
+id: creating-patches
+title: Creating patches
+---
+
+To make a contribution to another user's project, you must first find it on the
+network. To find a project, you need its [**Radicle ID**][ri]. This can be found
+on the project page under the project name, or in a seed node dashboard such as
+[sprout.radicle.xyz](http://sprout.radicle.xyz).
+
+![Identity][id]
+
+Once you have the Radicle ID, copy it into the Upstream search bar. From there,
+you can follow the project to track it. Once it is found in your network of
+connected peers, it will replicate the project to your machine. You won't see
+any information on the project until it is verified by the protocol.
+
+![Search bar][sb]
+
+![Found Project][fp]
+
+### Forking projects
+
+To contribute to another user's project you have to fork it forst.Using the
+**Fork** option in the Upstream client will clone the project to a specified
+location on your local machine and publish your version to the Radicle network
+as a new project.
+
+![Fork Project][fo]
+
+ Forking a project will move it from your **Following** tab to your **Projects**
+tab. This is because once you fork a project, you create your own view of it
+that is unique to your Device ID. You can [push changes][pc] to this view via
+the `rad` remote and broadcast them to your network of peers. You can clone your
+project with the **Checkout** option on your project page.
+
+### Creating a patch
+
+Once you've checked out your fork of the project you'd like to contribute to,
+you can **create a patch**. A patch is a set of proposed changes to the source
+code of a project. Patches can be accepted & merged or rejected by a project's
+maintainers. Similar to pull requests and merge requests, patches are used to
+manage contributions to code repositories.
+
+When viewing your fork of a project, you'll see a **Patches** tab next to
+**Files** and **Commits**. The **New Patch** button will guide you through the
+following steps:
+
+![New Patch][np]
+
+First, make your changes and commit them locally. Next, package them as a patch
```

Comment:
Maybe we can mention a branch explicitly and also use “mark” instead of “package”. The latter sounds like it’s a heavy operation.

```suggestion
First, make your changes and commit them locally to a branch. Next, mark them as a patch
```